package.json

```diff
@@ -40,22 +40,25 @@
     "build-electron-all": "npm run clean && npm run build-electron -- --platform=win32 && npm run build-electron -- --platform=linux && npm run build-electron -- --platform=darwin",
     "clean": "rm -rf builds config.json tasks.json coverage dist",
     "lint": "eslint *.js app test",
-    "dist": "npm run clean && electron-builder -wlm"
+    "dist": "npm run clean && electron-builder -wlm --x64 --ia32",
+    "dist-linux": "npm run clean && electron-builder -l --x64 --ia32",
+    "dist-win": "npm run clean && electron-builder -w --x64 --ia32",
+    "dist-mac": "npm run clean && electron-builder -m"
   },
   "build": {
     "appId": "angrykoala.gaucho",
     "linux": {
       "target": [
         "deb",
         "rpm",
-        "AppImage"
```

Comment:
Why did you remove AppImage build?

Reply:
It needed some metadata that wasn't present and kept failing.

Reply:
Is that error part of the errors in https://github.com/angrykoala/gaucho/wiki/Common-Issues ?

I didn't have that error, I would like to maintain AppImage support unless this is not fixable

Reply:
Nope, none of those. I could get it back and let you test it (if it worked for you previously, it should still work).

Reply:
Could you show me the error? (I already tested your version with AppImage and it seemed to work, I'll try again)

Reply:
It shows me this:
`./dist/__appimage-x64 should be packaged as /home/AJ/hacktoberfest/gaucho/dist/gaucho-0.4.0-x86_64.AppImage`